### PR TITLE
Multiple arguments in module:status command

### LINKED ISSUE #29344: [Issue] Multiple arguments in module:status command
This issue is automatically created based on existing pull request: magento/magento2#28250: Multiple arguments in module:status command

---------

### Preconditions: (*)

As of now `module:status` command accepts only one parram. But `module:enable` and `module:disable` accepts multiple module names as argument. So, in same way `module:status` command also can accept multiple module names as argument.

### Related Pull Requests
NA

### Steps to reproduce (*)

1. Run `module:status` command with multiple module names. ex `php bin/magento module:status Magento_Version Magento_Wishlist`.

**Actual Result:** :heavy_multiplication_x: Too many arguments....

Screenshot from 2020-07-31 13-26-53

**Expected Result:** :heavy_check_mark: We will get cli output in the following pattern.
`Magento_Version : Module is disabled`
`Magento_Wishlist : Module is enabled` 

Screenshot from 2020-07-31 13-24-17

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by chandru-199**
### Description (*)

As of now `module:status` command accepts only one parram. But `module:enable` and `module:disable` accepts multiple module names as argument. So, in same way `module:status` command also can accept multiple module names as argument.

### Related Pull Requests
NA

### Manual testing scenarios (*)

1. Run `module:status` command with multiple module names. ex `module:status module1 module2`.
2. We will get cli output in following pattern.
`module1 : Module is enabled`
`module2 : Module is disabled`

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#29344: Multiple arguments in module:status command

### CODE REVIEW

**Comment by chandru-199** [comment]
@magento run all tests

**Comment by chandru-199** [comment]
@magento run all tests

**Comment by chandru-199** [comment]
@magento run Static Tests

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi @chandru-199,
Could you fix static test failures?

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Comment by chandru-199** [comment]
@ihor-sviziev : sure.

**Comment by chandru-199** [comment]
Thank you @engcom-Charlie for you static test fix.

File: setup/src/Magento/Setup/Console/Command/ModuleStatusCommand.php [hunk 1]
```diff
@@ -40,26 +36,33 @@ public function __construct(ObjectManagerProvider $objectManagerProvider)
...
     {
-        $moduleName = (string)$input->getArgument('module');
-        if ($moduleName) {
-            return $this->showSpecificModule($moduleName, $output);
+        $moduleNames = $input->getArgument('module-names');
+        if (!empty($moduleNames)) {
+            foreach ($moduleNames as $moduleName) {
+                $this->showSpecificModule($moduleName, $output);
+            }
+            return 0;
```
Comment (ihor-sviziev) [inline setup/src/Magento/Setup/Console/Command/ModuleStatusCommand.php]:
```suggestion
            return Cli::RETURN_SUCCESS;
```

File: setup/src/Magento/Setup/Console/Command/ModuleStatusCommand.php [hunk 2]
```diff
@@ -79,34 +82,42 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln("<info>List of disabled modules:</info>");
         $this->showDisabledModules($output);
         $output->writeln('');
+
+        return 0;
```
Comment (ihor-sviziev) [inline setup/src/Magento/Setup/Console/Command/ModuleStatusCommand.php]:
```suggestion
        return Cli::RETURN_SUCCESS;
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Looks like we have few small issues. Please check them.

Everything else looks good to me

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7925](https://jira.corp.magento.com/browse/ENGCOM-7925) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
@magento create issue

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Manual testing scenario:**

Run `module:status` command with multiple module names. ex `php bin/magento module:status Magento_Version Magento_Wishlist`.

**Before:** :heavy_multiplication_x: Too many arguments....

Screenshot from 2020-07-31 13-26-53


**After:** :heavy_check_mark: We got CLI output in the following pattern.
`Magento_Version : Module is disabled`
`Magento_Wishlist : Module is enabled` 

Screenshot from 2020-07-31 13-24-17

**Case 2. With one non-existent module**

Run `module:status` command with multiple module names. ex `php bin/magento module:status Magento_Version Magento_Test`.

**After:** :heavy_check_mark: We got CLI output in the following pattern.

Screenshot from 2020-07-31 13-25-15

**Pull request merged**